src/components/layout/Container.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import Header from './header'
+
+const LeaderBoard = (props) => {
```

Comment:
@lovetrivedi Just to improve code consistency if we have only one parameter we will get rid of the parenthesis. So instead of `(props)` use `props`

Reply:
done